src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1260,16 +1260,11 @@ def test_WINNF_FT_S_FDB_8(self, config_filename):
     current_time = datetime.now(timezone(CPAS_TIME_ZONE))
     
     # Checks if CPAS start time is over then wait till next day otherwise
-    # wait till scheduled CPAS starts 
-    if current_time.hour < CPAS_START_TIME:
-      scheduled_cpas_start_time = current_time + timedelta(hours=CPAS_START_TIME - current_time.hour,
-                                                           minutes=-current_time.minute,
-                                                           seconds=-current_time.second)
-    else:
-      scheduled_cpas_start_time = current_time + timedelta(days=1, 
-                                                           hours=CPAS_START_TIME - current_time.hour,
-                                                           minutes=-current_time.minute,
-                                                           seconds=-current_time.second)
+    # wait till scheduled CPAS starts
+    scheduled_cpas_start_time = datetime.datetime.combine(datetime.date.today(), datetime.time(CPAS_START_TIME))
```

Comment:
I think you have too many `datetime`s here (the calls are not consistent with line 21). Please fix and check that the code can execute.

Reply:
Sorry, my bad. Fixed now.